src/components/chat/chatsnap/ChatSnap.tsx

```diff
@@ -87,10 +87,32 @@ const ChatSnap = ({ pfp, username, chatSnapMsg, timestamp, selected, onClick, is
     ) : null;
 
   // get date
+
+  const getAppropriateTimestamp = (time: Date) => {
```

Comment:
Why does this PR contains timestamp changes?

Reply:
I think by mistake this branch created from timestamp one